backend/api/views/model.py

```diff
@@ -25,6 +34,29 @@
 logger = structlog.get_logger(__name__)
 
 
+def _create(request, basename, get_success_headers):
+    """Create new models."""
+
+    registered_models = json.loads(request.data["metadata"])
+
+    data = []
+    for registered_model in registered_models:
+        registered_model["channel"] = get_channel_name(request)
+        serializer = ModelSerializer(data=registered_model)
+        try:
+            serializer.save_if_not_exists()
+        except AlreadyExistsError:
+            # May happen if the events app already processed the event pushed by the orchestrator
+            model = Model.objects.get(key=registered_model["key"])
+            serializer = ModelSerializer(model)
+        else:
+            data.append(serializer.data)
```

Comment:
this should be executed even if there's an `AlreadyExistsError` exception?

Reply:
Done [98271d8](https://github.com/Substra/substra-backend/pull/479/commits/98271d8fe81bbeef2824977eb750c0c1fd13dbab)